Sources/SnapshotTesting/Snapshotting/Any.swift

```diff
@@ -55,7 +55,12 @@ private func snap<T>(_ value: T, name: String? = nil, indent: Int = 0) -> String
 }
 
 private func sort(_ children: Mirror.Children) -> Mirror.Children {
-  return .init(children.sorted { snap($0) < snap($1) })
+  return .init(
+    children
+      .map({ (child: $0, snap: snap($0)) })
+      .sorted(by: { $0.snap < $1.snap })
+      .map({ $0.child })
```

Comment:
I wonder if we can simplify further if we:

1. Remove this `sort` function entirely
2. Change `var children = mirror.children` to `var children = mirror.children.map { snap($1, name: $0, indent: indent + 2) }`
3. Change `sort(children)` calls to `children.sort()`

That should eliminate some extra copies/allocations from the `map`s.